### ISSUE TITLE
Can't get Console Log 

### ISSUE THREAD

**Issue by ElitesTeaTime** [Kyle-Kyle/baidudl#160]
![asr](https://user-images.githubusercontent.com/44872257/48358487-762aca00-e6cd-11e8-8449-42a14bf94e1a.png)

I tried to get log why i can't get hlink. when i run program but log won't appeared. i updated to latest version.

**Comment by murath9** [comment]
i have the same problem :(

**Comment by Kyle-Kyle** [comment]
console log is in the web page nor in the pop page. But in the background page.
Steps to see console log:
->  open `chrome://extensions/`
->  click `background page` in baidu-dl

**Comment by jiangqinhua** [comment]
no speed at all even if I get the H link, is it normal ?

**Comment by jiangqinhua** [comment]
same issue here , Failed to load resource: the server responded with a status of 403 (Forbidden)

**Comment by Kyle-Kyle** [comment]
@jiangqinhua Yes. It's confirm that this problem happens on windows. I will have a look at it later.

**Comment by Superhackman** [comment]
@Kyle-Kyle I already found the problem in this case, the problem is in the cookies, its extension generates invalid links because it does not admit cookies and when you download them with idm, chrome or rpcdownload it gives an unauthorized user error because there are no cookies. , that there is not or does not detect any open session.
I am afraid that in its extension you must add internal support of download manager and support to store cookies and you can even make your extension stay with the session open when downloading

**Comment by Kyle-Kyle** [comment]
@Superhackman disagree? This extension uses the same cookie as Chrome itself uses.

**Comment by Superhackman** [comment]
@Kyle-Kyle okay it uses it and if in fact it generates valid links, but ... what happens that when using that link in another side like in idm, rpcdownload or even in the same browser chrome in another tab does not detect any open session of your account baidu cloud
that is, there must be a completely open session from the start of the download until it is finished

so it works pandownload, you start session from the same app pandownload and there are same files that you have and downloads without problems because when you download, baidu api when doing a get and post detects that there is an open session
